plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -126,6 +127,12 @@ public Builder setInitialTables(Iterable<TpchTable<?>> initialTables)
             return this;
         }
 
+        public Builder setSchemasLocationBase(String schemasLocationBase)
```

Comment:
This is applied only to tpch,tpch_bucketed schema, so name this properly.
(or make it applied universally to all CREATE SCHEMA, which would be even better!)

Reply:
I will name it `initialSchemasLocationBase` and keep it for tpch/tpch_bucketed for now. Can be improved later.